lib/index.js

```diff
@@ -13,12 +15,20 @@ const linterUiDefault = {
       atom.packages.getLoadedPackage('linter-ui-default').metadata['package-deps'].push('busy-signal')
     }
 
-    if (!atom.inSpecMode()) {
-      // eslint-disable-next-line global-require
-      require('atom-package-deps').install('linter-ui-default', true)
+    let callbackID
+    const installLinterUIDefaultDeps = () => {
```

Comment:
So it has a specific name in the profiler, remember that several packages will be deferring dependency installation till later so without a specific name it would be impossible to identify which was which.